CONTRIBUTORS.md

```diff
@@ -0,0 +1,26 @@
+# Contributors
+
+This project is the result of the dedicated efforts of many people. It is because of their generous contributions that open-source project like this one are alive.
```

Comment:
I think that would be the plural "open-source project**s**", right?

Reply:
Fixed